packages/web/src/common/store/pages/search-page/lineups/tracks/sagas.ts

```diff
@@ -97,11 +106,9 @@ function* getSearchPageResultsTracks({
 
       // getTracks returns an unsorted map of ID to track metadata.
       // We sort this object by trackIds, which is returned sorted by discprov.
-      const [tracks, sortedIds] = yield* all([
-        select(getTracks, { ids: trackIds }),
-        select(getSearchResultsPageTracks)
-      ])
-      const sortedTracks = (sortedIds as number[]).map((id) => tracks[id])
+      const tracks = yield* select(getTracks, { ids: trackIds })
```

Comment:
Please double check my thinking here. The previous logic made no sense to me because we select `trackIds` and then selected it again in the `all` call

Reply:
yea I think you're right, this is one less fetch